packages/libmikmod/SPECS/libmikmod.spec

```diff
@@ -0,0 +1,249 @@
+Summary:        A MOD music file player library
+Name:           libmikmod
+Version:        3.3.11.1
+Release:        7%{?dist}
+License:        GPLv2 and LGPLv2+
+URL:            http://mikmod.sourceforge.net/
+Source0:        http://downloads.sourceforge.net/mikmod/libmikmod-%{version}.tar.gz
+Patch0:         libmikmod-64bit.patch
+Patch1:         libmikmod-multilib.patch
+Patch2:         libmikmod-cflags.patch
+BuildRequires:  gcc
+BuildRequires:  alsa-lib-devel pulseaudio-libs-devel
+
+%description
+libmikmod is a library used by the mikmod MOD music file player for
+UNIX-like systems. Supported file formats include MOD, STM, S3M, MTM,
+XM, ULT and IT.
+
+%package devel
+Summary:        Header files and documentation for compiling mikmod applications
+Provides:       mikmod-devel = %{version}-%{release}
+#Requires:       %{name}%{?_isa} = %{version}-%{release}
```

Comment:
This one should normally stay (uncommented) I think - it's a dependency on the regular package on the arch from the development package.